SofaKernel/SofaCommon/config.h.in

```diff
@@ -94,7 +94,7 @@
 #  define SOFA_EIGEN2_SOLVER_API SOFA_IMPORT_DYNAMIC_LIBRARY
 #endif
 
-#cmakedefine01 SOFAEIGEN2SOLVER_HAVE_OPENMP
+#cmakedefine SOFAEIGEN2SOLVER_HAVE_OPENMP
```

Comment:
If you do this you won't be able to do `if(SOFAEIGEN2SOLVER_HAVE_OPENMP)` in the code.
Why then?

Reply:
@guparan Where have you seen `if(SOFAEIGEN2SOLVER_HAVE_OPENMP)` in the code?

Reply:
It is only used as a preprocessor token, as it should

 `#ifdef SOFAEIGEN2SOLVER_HAVE_OPENMP`